### DISCUSSION TITLE
Overlapping patches using Kornia

### DISCUSSION THREAD

**Discussion by kiratben** [kornia/kornia#2517] [Questions]
After using `extract_tensor_patches `to generate overlapping patches using a non-equal stride and window size,
`patches = kornia.contrib.extract_tensor_patches(x, window_size=(3, 3), stride=(1, 1))`
I’m attempting to reconstruct the tensor using `combine_tensor_patches`.
`output = kornia.contrib.combine_tensor_patches(patches, original_size=x.shape[2], window_size=(3,3), stride=(1,1))`
Unfortunately, I’m running into the following problem: `NotImplementedError: Only stride == window_size is supported. Got (1, 1) and (3, 3).` If anyone has encountered this before, could you please share a solution or suggest an alternative approach? Your insights would be greatly appreciated."

**Comment by edgarriba** [comment]
duplicated https://github.com/kornia/kornia/discussions/2513
